crates/iota-keys/src/keystore.rs

```diff
@@ -451,7 +452,8 @@ impl FileBasedKeystore {
                 .map_err(|e| anyhow!(e))?,
         )
         .with_context(|| format!("Cannot serialize keystore to file: {}", self.path.display()))?;
-        fs::write(&self.path, store)?;
+        fs::write(&self.path, store)
+            .map_err(|e| anyhow!("Couldn't save keystore to {}: {e:?}", self.path.display()))?;
         println!("Keys saved as Bech32.");
```

Comment:
bit annoying that those two methods { save_aliases, save_keystore } seem to be a bit inconsistent. Could we in both case just use either the plural { aliases, keys } or { aliasstore, keystore }?, also in line 457 there's this weird println! that's probably a debug remnant. Wdtyt?

Reply:
For the naming I don't care, feel free to open an issue for this
But I agree to other part, the print can be removed